build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "LibPQ"
+version = v"12.0.0+0"
+
+# Collection of sources required to build LibPQ
+sources = [
+    "https://ftp.postgresql.org/pub/source/v12.0/postgresql-12.0.tar.gz" =>
+    "15c7f267b476d764c79401d7f61f39c76222314951f77e6893a5854db26b6616",
+
+    "https://data.iana.org/time-zones/tzdata-latest.tar.gz" =>
+    "79c7806dab09072308da0e3d22c37d3b245015a591891ea147d3b133b60ffc7c",
+
+    "https://data.iana.org/time-zones/tzcode-latest.tar.gz" =>
+    "f6ebd3668e02d5ed223d3b7b1947561bf2d2da2f4bd1db61efefd9e06c167ed4",
```

Comment:
Do we want to just pick a specific version so this is always retro-actively reproducible

Reply:
Yes